third_party/2and3/redis/client.pyi

```diff
@@ -350,3 +354,9 @@ class Script:
     sha: Any
     def __init__(self, registered_client, script) -> None: ...
     def __call__(self, keys=..., args=..., client=...): ...
+
+class Monitor:
```

Comment:
Since the redis stubs support Python 2, this needs to derive from `object` to mark it as a new-style class:

```suggestion
class Monitor(object):
```